src/main/java/org/broadinstitute/hellbender/tools/AnalyzeSaturationMutagenesis.java

```diff
@@ -118,17 +120,22 @@
     @Argument(doc = "paired mode evaluation of variants (combine mates, when possible)", fullName = "paired-mode")
     private static boolean pairedMode = true;
 
+    @Argument(doc = "write BAM of rejected reads", fullName = "write-rejected-reads")
+    private static boolean writeRejectedReads = false;
+
     @VisibleForTesting static Reference reference;
     @VisibleForTesting static CodonTracker codonTracker; // device for turning SNVs into CodonVariations
+    @VisibleForTesting static SAMFileGATKReadWriter bamWriter;
```

Comment:
what about naming it something like `rejectedReadsBamWriter`, for clarity on purpose?

Reply:
done